WooCommerce/Classes/ViewRelated/Blaze/AdDestination/BlazeAdDestinationSettingView.swift

```diff
@@ -0,0 +1,222 @@
+import SwiftUI
+
+/// View to set ad destination for a new Blaze campaign
+struct BlazeAdDestinationSettingView: View {
+    @Environment(\.dismiss) private var dismiss
+
+    @ObservedObject private var viewModel: BlazeAdDestinationSettingViewModel
+
+    typealias DestinationType = BlazeAdDestinationSettingViewModel.DestinationURLType
+
+    init(viewModel: BlazeAdDestinationSettingViewModel) {
+        self.viewModel = viewModel
+    }
+
+    var body: some View {
+        NavigationView {
+            VStack(alignment: .leading, spacing: 0) {
+                // URL destination section
+                VStack(spacing: 0) {
+                    sectionHeading(title: Localization.destinationUrlHeading)
+                    destinationItem(title: Localization.productURLLabel,
+                                    subtitle: String(format: Localization.destinationUrlSubtitle, viewModel.productURL),
+                                    type: DestinationType.product,
+                                    showBottomDivider: true)
+
+                    destinationItem(title: Localization.siteHomeLabel,
+                                    subtitle: String(format: Localization.destinationUrlSubtitle, viewModel.homeURL),
+                                    type: DestinationType.home)
+                }
+                .padding(.bottom, Layout.sectionVerticalSpacing)
+
+                // URL parameters section
+                VStack {
+                    sectionHeading(title: Localization.urlParametersHeading)
+
+                    VStack {
+                        ForEach(viewModel.parameters, id: \.self) { parameter in
+                            parameterItem(itemName: parameter.key)
+                        }
+
+                        Button(Localization.addParameterButton) {
+                            // todo
+                        }
+                        .buttonStyle(PlusButtonStyle())
+                        .padding([.leading, .trailing], Layout.contentSpacing)
+                        .padding(.bottom, Layout.parametersVerticalSpacing)
+                    }
+                    .background(Color(.systemBackground))
```

Comment:
We should use `listForeground(modal: false)` for the foreground of lists.

Reply:
Thanks! TIL about `listForeground` and `listBackground` in general. This background setting is already refactored away in the next PR, so it is resolved.